FactoryMethod/Projects/Logistics/src/Destination.h

```diff
@@ -0,0 +1,33 @@
+//
+// Created by @PolinaShlepakova on 16.02.2019.
+//
+
+#ifndef LOGISTICS_DESTINATION_H
+#define LOGISTICS_DESTINATION_H
+
+#include <string>
+#include <iostream>
+
+class Destination {
+public:
+	Destination(const std::string& country, const std::string& city,
+				const std::string& addressLine, const std::string& index = "");
+	~Destination();
+	Destination(const Destination& other);
+	Destination& operator=(const Destination& other);
+
+	const std::string& country() const;
+	const std::string& city() const;
+	const std::string& addressLine() const;
+	const std::string& index() const;
+
+private:
+	std::string _country;
+	std::string _city;
+	std::string _addressLine;
```

Comment:
@PolinaShlepakova, do you mean street in `_addressLine` parameter?

Reply:
No, I meant exactly `_addressLine`.
The address line is a standard name for such information: street, house number and possibly apartment.